spec/e2e/unpersisted_association_spec.rb

```diff
@@ -237,7 +237,7 @@ def save_and_expect_rel!
           chris = Student.create(name: 'Chris', favorite_class: math)
           expect(chris.errors).to be_empty
 
-          lessons = chris.query_as(:c).match('c-[:FAVORITE_CLASS]->(l:Lesson)').pluck('l.uuid')
+          lessons = chris.query_as(:c).match('c-[:FAVORITE_CLASS]->(l:Lesson)').pluck(l: Lesson.id_property_name)
```

Comment:
Yeah, agreed that using `Lesson.id_property_name` is the right approach here.  I'm not 100% sure why this is using the `Query` API from `neo4j-core`, but I'm guessing it's to verify in the database without depending on other functionality of `neo4j`